apps/sms/views/conversation/js/compose.js

```diff
@@ -143,22 +138,19 @@ var Compose = (function() {
     state.emptyMessage = isEmptyMessage;
 
     Compose.updateEmptyState();
-    Compose.updateSendButton();
     Compose.updateType();
+    Compose.updateSendButton();
+    Compose.updateAttachButton();
```

Comment:
I'm trying to remember if it was important to call updateSendButton before updateType; I remember it took me some time to get the order right but I don't remember the issues I had...